moly-mofa/src/lib.rs

```diff
@@ -273,10 +274,12 @@ pub enum ChatResponse {
     ChatFinalResponseData(ChatResponseData),
 }
 
-// ====
-
 #[derive(Clone, Debug, Serialize, Deserialize)]
 struct ChatRequest {
     model: String,
     messages: Vec<MessageData>,
 }
+
+fn unique_agent_id(agent_id: &AgentId, server_address: &str) -> AgentId {
```

Comment:
> double thinking, this can even be the `new` constructor of `AgentId`
> 
> ```rust
> AgentId::new("whatever", "http://hmm.com");
> ```

Good catch! 🙌🏽